src/UberStrok.Realtime.Server.Game/Core/BaseGameRoom.cs

```diff
@@ -318,6 +318,11 @@ protected virtual void OnPlayerJoined(PlayerJoinedEventArgs args)
             PlayerJoined?.Invoke(this, args);
         }
 
+        protected virtual void OnPlayerLeft(PlayerLeftEventArgs args)

```

Comment:
I believe this function is never called, so the override in `DeathMatchGameRoom` is never called as well. It would probably be best to call it in `bool BaseGameRoom.DoDamage(...)` just after the player' state has been set to `PlayerState.Id.Killed`, [here](https://github.com/FICTURE7/uberstrok/blob/807e2515e6aabbc51898a839e40898eea8b34580/src/UberStrok.Realtime.Server.Game/Core/BaseGameRoom.cs#L304).

Reply:
Never mind, I thought this was PlayerKilled. :^{